### DISCUSSION TITLE
Component uses wrong color on first load/reload

### DISCUSSION THREAD

**Discussion by seveves** [vercel/next.js#25201] [Help]
I have a little site hosted here https://trails.naturfreunde-lichtenwald.de/ that is a next.js app hosted on vercel.
On the first load or any reload the color of the badge inside the card saying "Naddefatz" is yellow but when the trail is closed (which is for example in the evening) it should be red. The text on the card is updated correctly because it says that the trail is already closed.
So you can find the code for the TrailState component [here](https://github.com/seveves/nfliwa-trails/blob/main/components/TrailState.js). I'm not sure if I'm doing something you shouldn't do when using React or if it's some strange SSR behavior ... anyway, I don't know what I'm doing wrong. Please help :)

**Comment by bobarros** [comment]
@seveves,
I tried to reproduce the error but had no luck with it. Check it out:

https://next-js-testing-bobarros.vercel.app/

Maybe the problem is in part of the code that I can't access.
I'm also not able to realize the problem just by looking at your code. 
I had to mock/ignore data from here:

```jsx
import { renderMetaTags } from 'react-datocms';
import { TRAILS_QUERY } from '../queries/trails';
import { request } from '../lib/datocms';
```

I hope you find the problem.
